.env.example

```
@@ -81,6 +81,15 @@ ACCESS_API_KEY=ik_your-api-key-here-32-chars-minimum
 # Only needed if using cloud features
 CLOUD_API_HOST=https://api.insforge.dev
 
+# -----------------------------------------------------------------------------
+# Deployment Configuration (Optional)
+# -----------------------------------------------------------------------------
+# Required for self-hosted site deployments and custom domains.
+# Legacy deployments also require AWS_S3_BUCKET to be configured.
+VERCEL_TOKEN=
+VERCEL_TEAM_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VERCEL_TEAM_ID key should go before the VERCEL_TOKEN key

(UnorderedKey)